README.md

```diff
@@ -277,16 +277,18 @@ See [Config.kt](buildSrc/src/main/kotlin/Config.kt#L2txt) for the latest version
 
 # Kotlin Memory Model and Coroutine compatibility
 
-Realm Kotlin is implemented against Kotlin's default memory model (the old one), but still supports running with the new memory model if enabled in the consuming project. See https://github.com/JetBrains/kotlin/blob/master/kotlin-native/NEW_MM.md#switch-to-the-new-mm for details on enabled the new memory model.
+Realm Kotlin 1.3.0 and above *only* works with the new Kotlin Native memory model. This is also the default memory model from Kotlin 1.7.20 and onwards. This mean that you need the default Kotlin Coroutine library 1.6.0 and above and not the `-mt` variant, which have also been [deprecated](https://blog.jetbrains.com/kotlin/2021/12/introducing-kotlinx-coroutines-1-6-0/). 
```

Comment:
Should we hint to the other branch/version one should use if we want to continue working with the old memory model (i.e the Jetpack Compose stable)

Reply:
I think we agreed there was no need to mention other releases until we actually needed them. Since Compose right now supports 1.7.20, they don't actually need to care about the difference.